### ISSUE TITLE
Parsing failures

### ISSUE THREAD

**Issue by ekutner** [HandyGuySoftware/dupReport#174]
As far as I can tell, parsing of the source and destination from the email subject line, as well as parsing the values from the body of the email, when sent in Duplicati format, doesn't work.
I have a fix for both issues but since the request is to discuss before posting a pull request I'd like to verify that this issue is affecting others as well.

**Comment by HandyGuySoftware** [comment]
@ekutner , Thanks for sending in your comment. It's very interesting that the parsing routines don't work properly for you. The program has been in production use for a number of years, and I use it personally, so I know the parsing routines work, at least in those cases. I also know it works for both Duplicati and JSON email formats. 

However, if it's not working for you let's try to figure out why. Perhaps there is something about your specific email server that may be causing an issue? Some questions to consider:

- Are you getting a specific error message that may indicate the problem or does the program just not pick up any emails?
- If you run the full debug log (-v7 command line option), does the log show an indication of why it isn't working properly?

Because this is the first report of this issue in the latest version of the program I'd like to track down the root cause in the existing code, with your assistance. In addition, if you think your proposed fix would improve the program and make it more stable, I'd be interested in seeing your code. I have created a new branch ([Issue_174](https://github.com/HandyGuySoftware/dupReport/tree/Issue_174)) where you can issue your pull request.

HG

**Comment by HandyGuySoftware** [comment]
Also, would it be possible to post a copy of the verbose log (-v7) here so I can try to see what's going on?

**Comment by ekutner** [comment]
I don't have the log because it's already fixed for me. I can roll back my changes and get you the log but I think if you check out my PR it should be clear what I'm talking about.

1. The code for extracting the source and destination computers was pretty convoluted but more importantly, I can't see how it could have worked. The regex was only for the source+separator / dest but the string it operated on contained the whole subject line. It also double escaped a backslash character that was in the generated .rc file for the delimiter regex. The bottom line was that nothing was captured.  I changed that and also added support for a separator with whitespaces around it so it would be possible to create nicer looking backup names. My current .rc setting are:
srcregex = [^-]*
destregex = .*
srcdestdelimiter = " - "
2. The field value extraction assumes every line ends with a newline but, at least in my case, it was one big HTML with no line breaks and only a &lt;br/> tag to mark the end of each line. It can be a gmail thing but I doubt Google would change the contents of the email. It is also possible that the email has two embedded MIME types, one for text and one for html and you were getting the text version while I'm getting the html. Anyway, my fix should still work with text because it just replaces the BR tags with newlines.
3. There was a small typo in options.py that the IDE alerted me about so I fixed it too, but didn't actually hit it while running the program.
